pyproject.toml

```diff
@@ -82,6 +82,7 @@ ignore = [
     "SIM116", # Disable Use a dictionary instead of consecutive `if` statements
     "SIM117",
     "SIM118",
+    "Q000",  # please remove after ufmt is enabled globally
```

Comment:
[bad-quotes-inline-string (Q000)](https://docs.astral.sh/ruff/rules/bad-quotes-inline-string/#bad-quotes-inline-string-q000) is not enabled duo to too many changes.

Reply:
I feel like Q003/Q004 could be useful but why Q001/Q002?

Reply:
Do you concerning about `ruff` Q001/Q002 can conflict with `black`; or enabling these two rules is redundant?

Reply:
Both

Reply:
Removed Q000, Q001, and Q003. Leaving them to `black`.